ConfigurationSystem/scripts/dirac-admin-check-config-options.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+"""Check configuration options against the defaults in the ConfigTemplate.cfg files."""
```

Comment:
Maybe you add a bit more of info of when, why and by who this script should be used.